src/features/game/types/withdrawables.ts

```
@@ -1596,6 +1596,14 @@ export const INVENTORY_RELEASES: InventoryReleases = {
     tradeAt: CHAPTERS["Crabs and Traps"].endDate,
     withdrawAt: new Date("2026-06-04T00:00:00Z"),
   },
+    "Giant Isopod": {
```

ESLint

[error] Delete `··`

(prettier/prettier)